<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1667</width>
    <height>946</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(132, 132, 132);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="Tab">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>282</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(76, 76, 85);
background-image: url(:/design/Pictures/Accessory/fond.png);
color: rgb(85, 0, 127);
</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="A">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>     A     </string>
     </attribute>
    </widget>
    <widget class="QWidget" name="B">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>     B     </string>
     </attribute>
    </widget>
    <widget class="QWidget" name="C">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>     C     </string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Map">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>720</y>
      <width>281</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(81, 81, 81);</string>
    </property>
    <property name="title">
     <string>Carte</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>251</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="VisibleGrid">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::PreventContextMenu</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Quadrillage</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="NumberXText">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Nombre X de carreaux  :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="X">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="NumberYText">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Nombre Y de carreaux  :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="Y">
          <property name="styleSheet">
           <string>color: rgb(255, 255, 255);</string>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1591</width>
      <height>831</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="MainHorizontalLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1667</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <widget class="QMenu" name="File">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>FICHIER</string>
    </property>
   </widget>
   <widget class="QMenu" name="Edit">
    <property name="title">
     <string>EDITION</string>
    </property>
   </widget>
   <widget class="QMenu" name="Help">
    <property name="title">
     <string>AIDE</string>
    </property>
   </widget>
   <addaction name="File"/>
   <addaction name="Edit"/>
   <addaction name="Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(195, 195, 195);

</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="RegisterAction"/>
   <addaction name="separator"/>
   <addaction name="ImportAction"/>
   <addaction name="separator"/>
   <addaction name="ThirdPlaneAction"/>
   <addaction name="SecondPlaneAction"/>
   <addaction name="FirstPlaneAction"/>
   <addaction name="AjustOpacityAction"/>
   <addaction name="separator"/>
   <addaction name="EraserAction"/>
   <addaction name="PenAction"/>
   <addaction name="ColisionAction"/>
   <addaction name="separator"/>
   <addaction name="ScriptEditAction"/>
   <addaction name="separator"/>
   <addaction name="PlayAction"/>
   <addaction name="separator"/>
   <addaction name="CalculatorAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="RegisterAction">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/icons/Pictures/Icons/Enrengistrer.png</normaloff>:/icons/Pictures/Icons/Enrengistrer.png</iconset>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Register your project&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="CalculatorAction">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/icons/Pictures/Icons/calculette.png</normaloff>:/icons/Pictures/Icons/calculette.png</iconset>
   </property>
   <property name="text">
    <string>Calculator</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open the calculator&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="ImportAction">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/icons/Pictures/Icons/Importer.png</normaloff>:/icons/Pictures/Icons/Importer.png</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Import a picture&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="FirstPlaneAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/icons/Pictures/Icons/plan1.png</normaloff>:/icons/Pictures/Icons/plan1.png</iconset>
   </property>
   <property name="text">
    <string>First plane</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Access to first plane&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="SecondPlaneAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/icons/Pictures/Icons/plan2.png</normaloff>:/icons/Pictures/Icons/plan2.png</iconset>
   </property>
   <property name="text">
    <string>Second plane</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Access to second plane&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="ThirdPlaneAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/icons/Pictures/Icons/plan3.png</normaloff>:/icons/Pictures/Icons/plan3.png</iconset>
   </property>
   <property name="text">
    <string>Third plane</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Acess to third plane&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="EraserAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/icons/Pictures/Icons/gomme.png</normaloff>:/icons/Pictures/Icons/gomme.png</iconset>
   </property>
   <property name="text">
    <string>Eraser</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use the eraser&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="PenAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/icons/Pictures/Icons/crayonPapierIcone.png</normaloff>:/icons/Pictures/Icons/crayonPapierIcone.png</iconset>
   </property>
   <property name="text">
    <string>Pen</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use the pen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="ColisionAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/icons/Pictures/Icons/collision.png</normaloff>:/icons/Pictures/Icons/collision.png</iconset>
   </property>
   <property name="text">
    <string>Colision</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the colisions&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="AjustOpacityAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/icons/Pictures/Icons/oeil.png</normaloff>:/icons/Pictures/Icons/oeil.png</iconset>
   </property>
   <property name="text">
    <string>Opacity</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;To ajust planes opacity&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="ScriptEditAction">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/icons/Pictures/Icons/script.png</normaloff>:/icons/Pictures/Icons/script.png</iconset>
   </property>
   <property name="text">
    <string>Script</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open the script editor&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="PlayAction">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/icons/Pictures/Icons/jouer.png</normaloff>:/icons/Pictures/Icons/jouer.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Launch the game&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
